packages/opentelemetry-core/test/common/anchored-clock.test.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import assert = require('assert');
+import { performance } from 'perf_hooks';
+import { AnchoredClock } from '../../src';
```

Comment:
This should make the tests happy again. :slightly_smiling_face: 

```suggestion
import {
  AnchoredClock,
  otperformance
} from '../../src';
```